.release-it.json

```diff
@@ -3,14 +3,11 @@
     "release": true
   },
   "plugins": {
-    "@release-it/keep-a-changelog": {
-      "filename": "CHANGELOG.md",
-      "addUnreleased": true,
-      "strictLatest": false,
-      "addVersionUrl": true
-    }
+    "abeforgit/changesets-release-it-plugin": {}
```

Comment:
```suggestion
    "changesets-release-it-plugin": {}
```

Reply:
oh wow yeah how did that even work in the first place